src/praisonai-bot/praisonai_bot/gateway/push_delivery.py

```
@@ -186,6 +317,13 @@ async def purge_acknowledged(self, max_age_seconds: int = 86400) -> int:
                 del self._message_store[eid]
                 purged += 1
 
+        # Mirror the eviction to the durable store so it does not grow unbounded.
+        if self._sqlite_store is not None:
+            try:
+                self._sqlite_store.evict_expired(cutoff, keep=pending_ids)
+            except Exception as e:  # pragma: no cover - defensive
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)